lib/gstreamer/vppbase.py

```diff
@@ -116,6 +116,11 @@ def validate_caps(self):
     if self.mformat is None:
       slash.skip_test("{gstvpp}.{format} unsupported".format(**vars(self)))
 
+    if "sharpen" == self.vpp_op:
+      self.ihwformat = self.map_formatu(self.ifmt if self.ifmt in ifmts else None)
+      if self.ihwformat is None:
+        slash.skip_test("{ifmt} unsupported".format(**vars(self)))
+
```

Comment:
This is not needed.  Please remove this code change.

If the HW does not support the format directly, then the test will automatically insert compatible SW conversions into the pipeline.